teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<ul><li>Can you add links to the databases you mention here?</li><li>Maybe I'm misunderstanding sth here, but why is 'low quality' data used for classification?</li><li>Can you add some numbers about the size of the data set?</li><li>'The bioactivity data aggregated is standardized, repaired, and normalised'</li><li class="ql-indent-1">The aggregated bioactivity data ....</li><li class="ql-indent-1">And what exactly is meant by 'repaired'?</li><li>maybe shortly comment on what pChEMBL values are</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Please, let me know if the level of detail is good now. </p>